docs/source/scribe_data/cli.rst

```diff
@@ -1,10 +1,10 @@
-cli/
+Command Line Interface(CLI)
```

Comment:
Reverting this back as we want the title to reflect the project structure (changing this also changes the link to this page in the docs, and all the others are directories).

Reply:
Sure